sys/rman.c

```diff
@@ -0,0 +1,114 @@
+#include <rman.h>
+#include <stdc.h>
+#include <pool.h>
+
+static pool_t P_RMAN;
+
+static resource_t *find_resource(rman_t *rm, rman_addr_t start, rman_addr_t end,
```

Comment:
Missing documentation.

Reply:
I think that purpose of rman_create and find_resource are obvious, and comment would not tell much more, than signature tells. I've added comments to other functions.

Reply:
Ok.